lib/rules/length-zero-no-unit/index.js

```diff
@@ -29,6 +29,10 @@ const rule = function(actual, secondary, context) {
 		}
 
 		root.walkDecls((decl) => {
+			if (decl.prop.toLowerCase() === 'line-height') {
```

Comment:
Not only `line-height` can contains `<length> | <percentage>`, for example https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-width, https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dashoffset and other, you can find them here https://github.com/mdn/data/blob/master/css/properties.json, also let's put them in https://github.com/stylelint/stylelint/tree/master/lib/reference

Reply:
To be honest I completely lost the idea why I'd ignored ```line-height``` 😕 .
This is the same:
```css
line-height: 0;
line-height: 0px;
line-height: 0em;
```
So in case of zero value the dimension unit is useless and the warning should be flagged (or it should be auto-fixed).
The only unit to ignore here is ```fr``` since ```0fr``` and ```0``` mean completely different things.

Reply:
`line-height: 0;` !== `line-height: 0px;`

Сlarifications and examples - https://github.com/cssnano/cssnano/issues/768

Reply:
Make sense.
But what the point to ignore ```stroke-width``` for example. It doesn't matter whether it has unit or not. Example: https://jsfiddle.net/vc2otk06/

Reply:
Oh, yes, my mistake, you are right, don't found other properties with same ability as `line-height` has, let's add tests for this cases:
```css
a { font: normal normal 400 16px/0px cursive; }
```
```css
a { font: normal normal 400 16px/0 cursive; }
```

Reply:
The funny thing is these cases don't break tests in master branch but they have to.
I add some logic to process ```font``` declaration and update test cases.